src/modules/facility-loan-transaction/facility-loan-transaction.service.get-loan-transactions.test.ts

```diff
@@ -0,0 +1,170 @@
+import { BadRequestException } from '@nestjs/common';
+import { PROPERTIES } from '@ukef/constants';
+import { AcbsAuthenticationService } from '@ukef/modules/acbs-authentication/acbs-authentication.service';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import { getMockAcbsAuthenticationService } from '@ukef-test/support/abcs-authentication.service.mock';
+import { GetFacilityLoanTransactionGenerator } from '@ukef-test/support/generator/get-facility-loan-transaction-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { AcbsFacilityLoanTransactionService } from '../acbs/acbs-facility-loan-transaction.service';
+import { GetFacilityLoanTransactionResponseItem } from './dto/get-loan-transaction-response.dto';
+import { FacilityLoanTransactionService } from './facility-loan-transaction.service';
+
+describe('FacilityLoanTransactionService', () => {
+  const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;
```

Comment:
Can we destructure this?

Reply:
Changed to `const { portfolioIdentifier } = PROPERTIES.GLOBAL;`